packages/playwright/src/reporters/teleEmitter.ts

```diff
@@ -184,6 +184,7 @@ export class TeleReporterEmitter implements ReporterV2 {
       dependencies: project.dependencies,
       snapshotDir: this._relativePath(project.snapshotDir),
       teardown: project.teardown,
+      use: project.use,
```

Comment:
project.use can contain complex live objects, so we can't pass it w/o processing. I'd suggest that we cherry-pick what we want explicitly.